runhouse/resources/hardware/on_demand_cluster.py

```diff
@@ -615,3 +615,24 @@ def ssh(self, node: str = None):
                 return_cmd=True,
             )
             subprocess.run(cmd, shell=True)
+
+    @classmethod
+    def _load_from_local(cls, name: str):
+        """Try loading the cluster from data stored in the local Sky DB"""
+        state = sky.status(cluster_names=[name.split("/")[-1]], refresh=True)
+
+        if len(state) == 0:
+            raise ValueError(f"Cluster {name} not found locally")
+
+        resource_handle = state[0].get("handle", {})
+
+        # Create the Runhouse cluster config using the local Sky DB
+        config = {
```

Comment:
@dongreenberg thoughts on expanding this to match the full config that we store in Den? (ex: i want to save a cluster in Den that previously was only local)